File: 240712/사칙연산 함수/quadratic-operations-function.py
def plus(a, c):
    print(f'{a} + {c} = {a+c}')

def minus(a,c):
    print(f'{a} - {c} = {a-c}')

def multiple(a,c):
    print(f'{a} * {c} = {a*c}')

def divide(a,c):
    print(f'{a} / {c} = {a/c}')

a,o,c = tuple(input().split())
a = int(a)
c = int(c)
if o == '+':
    plus(a,c)
elif o == '-':
    minus(a,c)
elif o == '*':
    multiple(a,c)
elif o == '/':
    divide(a,c)
else:
    print('False')

File: 250114/사칙연산 함수/quadratic-operations-function.java
import java.util.Scanner;

public class Main {
    public static void plus(int a, int b) {
        System.out.printf("%d + %d = %d", a,b,a+b);
    }

    public static void minus(int a, int b) {
        System.out.printf("%d - %d = %d", a,b,a-b);
    }

    public static void multiply(int a, int b) {
        System.out.printf("%d * %d = %d", a,b,a*b);
    }

    public static void divide(int a, int b) {
        System.out.printf("%d / %d = %d", a,b,a/b);
    }

    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        char n = sc.next().charAt(0);
        int b = sc.nextInt();

        if(n=='+') {
            plus(a,b);
        }
        else if(n=='-') {
            minus(a,b);
        }
        else if(n=='*') {
            multiply(a,b);
        }
        else if(n=='/') {
            divide(a,b);
        }
        else {
            System.out.print("False");
        }


    }
}